internal/test/smoke/suite_base_test.go

```diff
@@ -135,8 +137,10 @@ func (s *SuiteBase) SetupTest() {
 		panic(err)
 	}
 	rbac := client_rbac.New(s.cfg.Clients.RbacBaseURL, rbacClient)
-	pendo := client_pendo.NewClient(s.cfg)
-	s.svc = service_impl.NewApplication(ctx, s.wg, s.cfg, s.db, inventory, rbac, pendo)
+	if s.PendoClient == nil {
+		s.PendoClient = client_pendo.NewClient(s.cfg)
+	}
```

Comment:
```suggestion
	s.PendoClient = mock_pendo.NewPendo(t)
```

I would suggest to use the mock generated by mockery tool, and this would make it available for any smoke test.

Reply:
Do we want to use the mock in all smoke tests?  The current implementation allows to choose. So if the mock is not se then it will use the `NewClientDump()` or real `NewClient()`.

Reply:
thanks, I didn't see in that way! I agree with the current change!